recipes/pyfaidx/0.4.7.1/build.sh

```diff
@@ -1,9 +0,0 @@
-#!/bin/bash
```

Comment:
why have you removed this?

Reply:
Hi, 'pyfaidx' is not the recipe I want to add. I have made some changes according to your commits, could you please review again?